hello_mina.py

```diff
@@ -0,0 +1 @@
+print("mina says hi")
```

Comment:
```suggestion
print("Mina says hi")
```
Proper nouns should be capitalized

Reply:
discord